apps/publisher/modules/lifecycle/lifecycle-api.js

```diff
@@ -334,7 +393,9 @@ var error = '';
             state.isDeletable = isDeletable(lcState, state.deletableStates, lcName);
         }
         //Update the state of the check items
-        state.checkItems = setCheckItemState(state.checkItems, asset, am, lcName);
+        state.checkItems = setCurrentCheckItemState(state.checkItems, lcCheckedStates);
+        state.approvedActions = new Object();
```

Comment:
Fixed in commit 86e200939329fb59ede32b4bcdea56a8a9db2abd [1]

[1] https://github.com/sameerak/carbon-store/commit/86e200939329fb59ede32b4bcdea56a8a9db2abd